File: 基礎題庫/Contents/e976/e976.c
#include<stdio.h>

int main(void)
{
	int h, m, s;
	
	while(scanf("%d %d %d", &h, &m, &s) != EOF)
	{
		if(h * s >= m)
			printf("%d %d %d. I will make it!\n", h, m, s);
		else
			printf("%d %d %d. I will be late!\n", h, m, s);
	}
	
	return 0;
}

File: 基礎題庫/Contents/e976/e976.py
import sys

for temp in sys.stdin:
    try:
        h, m, s = map(int, temp.split())
    except:
        continue

    if h * s >= m:
        print(h, m, str(s) + ".", "I will make it!")
    else:
        print(h, m, str(s) + ".", "I will be late!")
